tools/generate-documentation

```diff
@@ -83,7 +83,8 @@ update_docs() {
         mkdir out || exit 1
         (
         cd out
-        if [[ -z "${PULL_REQUEST_USER}" ]]; then
```

Comment:
But we're using Bash here and using `[[` avoids spawning a sub process so I wouldn't change that. The same counts for the other occurrences,